apps/dashboard/app/(app)/authorization/roles/[roleId]/delete-role.tsx

```
@@ -65,6 +65,11 @@ export const DeleteRole: React.FC<Props> = ({ trigger, role }) => {
     },
   });
 
+  function handleDialogOpenChange(newState: boolean) {
```

Biome

[error] This function is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.

(lint/correctness/noUnusedVariables)